contracts/InterchainTokenService.sol

```diff
@@ -387,6 +417,9 @@ contract InterchainTokenService is
         } else if (trustedAddressHash(chainName()) == ITS_HUB_ROUTING_IDENTIFIER_HASH) {
             // Currently, deployments directly on ITS contract (instead of ITS Factory) are restricted for ITS contracts deployed on Amplifier, i.e registered with the Hub
             revert NotSupported();
+        } else if (bytes(destinationChain).length == 0) {
```

Comment:
I looked a bit at explorers. Doesn't seem like any partners are using this method directly vs ITS factory. Let's gate it to be only callable via ITS factory?

Reply:
Yep, can you drop the ability in a follow up? We can take a look at the ITS token configs registered on squid, but don't think I see anyone using this